src/libraries/System.IO.Compression.ZipFile/tests/ZipFile.Unix.cs

```diff
@@ -184,9 +184,9 @@ public void UnixExtractFilePermissionsCompat(string zipName, string expectedPerm
         }
 
         [Fact]
-        [PlatformSpecific(TestPlatforms.AnyUnix & ~TestPlatforms.Browser & ~TestPlatforms.tvOS & ~TestPlatforms.iOS)]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
         [SkipOnPlatform(TestPlatforms.LinuxBionic, "Bionic is not normal Linux, has no normal file permissions")]
-        public async Task CanZipNamedPipe()
+        public void ZipNamedPipeIsNotSupported()
```

Comment:
This test previously showed a named pipe can be zipped, but it would hang if either the reader or writer Task is not there.

It's not likely the caller of `Directory.CreateDirectory` will orchestrate with the writer of the pipe to provide the data in the way this test did.

Reply:
Good catch. And thanks for enabling tvOS and iOS again. We just need to execute runtime-extra-platforms so they get tested.